modules/installation-osp-kuryr-octavia-upgrade.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-configure_{context}"]
+= Using the Octavia OVN load balancer provider driver with Kuryr SDN
+
+If your {product-title} cluster uses Kuryr and was installed on a {rh-openstack-first} 13 cloud
+that was later upgraded to {rh-openstack} 16, you can configure it to use the Octavia OVN provider driver.
+
+[CAUTION]
+====
+Kuryr replaces existing load balancers after you change provider drivers. This process
+results in some downtime.
+====
+
+.Prerequisites
+
+* Install the {rh-openstack} CLI, `openstack`.
+
+* Install the {product-title} CLI, `oc`.
+
+* Verify that the Octavia OVN driver on {rh-openstack} is enabled.
++
+[TIP]
+====
+To view a list of available Octavia drivers, on a command line, enter link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/command_line_interface_reference/loadbalancer#loadbalancer_provider_list[`openstack loadbalancer provider list`].
+
+You should see the `ovn` driver in the command's output.
+====
+
+.Procedure
+
+To change from the Octavia Amphora provider driver to Octavia OVN:
+
+. Open the `kuryr-config` ConfigMap. On a command line, enter:
++
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+----
+
+. In the ConfigMap, delete the line that contains `kuryr-octavia-provider: default`. For example:
++
+----
+...
+kind: ConfigMap
+metadata:
+  annotations:
+    networkoperator.openshift.io/kuryr-octavia-provider: default <1>
+...
+----
+<1> Delete this line. The cluster will regenerate it with `ovn` as the value.
++
+Wait for the Cluster Network Operator detects the modification and redeploys the `kuryr-controller` and `kuryr-cni` pods. This process may take several minutes.
+
+. Verify that the `kuryr-config` ConfigMap annotation is present with `ovn` as its value. On a command line, enter:
++
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+----
++
+The `ovn` provider value should be present in the output:
++
+----
+...
+kind: ConfigMap
+metadata:
+  annotations:
+    networkoperator.openshift.io/kuryr-octavia-provider: ovn
+...
+----
+
+. Verify that {rh-openstack} recreated its load balancers.
+
+.. On a command line, type:
++
+----
+$ openstack loadbalancer list | grep amphora
+----
++
+You should see a single Amphora load balancer. For example:
++
+----
+a4db683b-2b7b-4988-a582-c39daaad7981 | ostest-7mbj6-kuryr-api-loadbalancer  | 84c99c906edd475ba19478a9a6690efd | 172.30.0.1     | ACTIVE              | amphora
+----
+
+.. Then, type:
```

Comment:
A little context to this step might read better. For example, "Search for `ovn` loadbalancers by entering:"